tests/test_pylint_runners.py

```diff
@@ -53,16 +54,16 @@ def test_pylint_run_jobs_equal_zero_dont_crash_with_cpu_fraction(
     """
     builtin_open = open
 
-    def _mock_open(*args, **kwargs):
+    def _mock_open(*args: str, **kwargs) -> BufferedReader:
         if args[0] == "/sys/fs/cgroup/cpu/cpu.cfs_quota_us":
             return mock_open(read_data=b"-1")(*args, **kwargs)
         if args[0] == "/sys/fs/cgroup/cpu/cpu.shares":
             return mock_open(read_data=b"2")(*args, **kwargs)
-        return builtin_open(*args, **kwargs)
+        return builtin_open(*args, **kwargs)  # type: ignore[call-overload]
```

Comment:
isn't this a real issue?

Reply:
I added back ``typing.Any`` to fix this, in this case we really don't want to detail the full typing of ``builtin.open``.